engine/core/src/main/java/org/datacleaner/job/builder/AbstractComponentBuilder.java

```diff
@@ -472,6 +472,18 @@ private void synchronizeDependentProperties(final ConfiguredPropertyDescriptor p
         }
     }
 
+    private static String getKey(final Object object) {
+        if (object instanceof InputColumn<?>) {
+            final InputColumn<?> inputColumn = (InputColumn<?>) object;
+
+            if (inputColumn.isVirtualColumn()) {
+                return inputColumn.getName();
```

Comment:
Is it the following case? I used the same name for both concatenators' outputs and there is a message "not unique columns". 
![same-input-column-names](https://cloud.githubusercontent.com/assets/13213915/19269294/15821602-8fba-11e6-8a84-e41ed062af80.png)